config/sync/upd_featured_case_studies.settings.yml

```diff
@@ -0,0 +1,5 @@
+featured_case_study1: '25'
+featured_case_study2: '13'
+featured_case_study3: '12'
```

Comment:
It would yes. If you don't want that to happen, the quick workaround would be add those to the config_ignore settings. A more proper solution would be to use the State API instead of config.